modules/odc-invoke-serverless-function.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="odc-invoke-serverless-function_{context}"]
+= Testing a function in the web console
+
+You can test a deployed serverless function by invoking it in the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on your {product-title} cluster.
+* You have logged in to the web console and are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created and deployed a function.
+
+.Procedure
+
+. In the *Developer* perspective, navigate to *Topology*.
+. Click on a function, then click *Test Serverless Function* from the *Actions* drop-down list in the *Details* panel. This opens the *Test Serverless Function* dialog box.
+
+. In the *Test Serverless Function* dialog box, modify the settings for your test as required:
+
+.. Choose the *Format* for your test. This can be either *CloudEvent* or *HTTP*.
+.. The *Content-Type* defaults to your function type.
```

Comment:
Actually, I believe it defaults to `application/json`. This is the `Content-Type` HTTP header value.